### ISSUE TITLE
[MAS4.2.10][Screen Reader-Open bot dialog] Keyboard focus and voiceover focus is not synchronized.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1889]
**Actual Result:**
When user navigates in the voiceover mode, the keyboard focus and voiceover focus is not synchronized at "Buttons" and "Edit fields".

**Expected Result:**
The focus should be synchronized with voice over focus so that all the changes are done by user itself.

**Note:**

1. Same issue found on "My Bots" section. {Refer attachment} (Scenario: "Welcome Tab screen") .
2. Same issue found on "How to build a Bot" section. {Refer attachment} (Scenario: How to Build a Bot".
3. Same issue found on "Connect your Bot to LUIS Application" dialog.{Refer attachment} (Scenario: Add a LUIS app manually)
4. Same issue found at "Browse" button as "Microsoft App Id" and "Microsoft App Password" edit fields in "Open a Bot" dialog. {Refer attachment} (Scenario: Open Bot Dialog)
5. Same issue found at "Microsoft App Id" and "Microsoft App Password" edit fields in "New bot configuration" dialog. {Refer attachment} (Scenario: Create new bot configuration) 
6. Same issue found on the edit fields present at "Settings" and "Sign in with Azure account dialog". {Refer attachment} (Scenario: Settings and Sign in with Azure account dialog)
7. Same issue found at link present at "Add language understanding" and "Add QnA maker" pop-up. {Refer attachment} (Scenario: Add language understanding and Add QnA maker)

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot Dialog" button and activate it.
4. Open Bot Dialog box gets opened.
5. Navigate to the dialog box in voiceover mode 
6. Observe the issue.

**User Impact:**
The voiceover focus and keyboard focus should be synchronized so that all the changes done in the pane is knew by the user. Accidentally changes in the pane can be prevented.

****Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}]

**Attachment:**
[Focus not synchronized.zip](https://github.com/microsoft/BotFramework-Emulator/files/3676304/Focus.not.synchronized.zip)

**Comment by tonyanziano** [comment]
Can this scenario be re-evaluated using the newest nightly version `4.6.0-x`?

I'm trying to reproduce this issue on the latest version, using Mac 4.13 and can't reproduce most of these Voiceover / keyboard focus synchronization issues.

The only issue that I am still seeing is with the "Browse button" in the Open Bot dialog, but other than that, I'm experiencing none of these issues.

Examine the below gif:

![out](https://user-images.githubusercontent.com/3452012/69279503-5ad48380-0b99-11ea-9930-f12fd9d3de97.gif)

**Comment by corinagum** [comment]
I am not able to reproduce this issue either. :)

**Comment by tonyanziano** [comment]
Nevermind my earlier comment. I tried using VoiceOver navigation (Ctrl + option + arrow) and was able to reproduce the issue.

**Comment by ShikhaMishra11** [comment]
@tonyanziano The issue still exist in latest build. Yes you can repro this, in voice over mode (control+option+left/right key).

Please find the latest attachment.
[Focus not synchronized 23Nov.zip](https://github.com/microsoft/BotFramework-Emulator/files/3882402/Focus.not.synchronized.23Nov.zip)

Test Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (version: 4.6.0-92058)

**Comment by ShikhaMishra11** [comment]
As we have checked and verified, the issue gets fixed in the latest version of BOT. So, we are closing the issue.

Regression Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.6.0-93410)
